src/Microsoft.AspNetCore.Server.IntegrationTesting/xunit/SkipIfCurrentRuntimeIsCoreClrAttribute.cs

```diff
@@ -17,7 +17,7 @@ public bool IsMet
         {
             get
             {
-                return !Process.GetCurrentProcess().ProcessName.ToLower().Contains("coreclr");
+                return !Process.GetCurrentProcess().ProcessName.ToLower().Contains("dotnet");
```

Comment:
Ah that would be better. I was wondering why this looks at the process name but I guess it's just really old code.